submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/mapping/SubmissionDbMapper.kt

```diff
@@ -76,27 +76,25 @@ class SubmissionDbMapper {
 }
 
 private class DbSectionMapper {
-    internal fun toSection(sectionDb: SectionDb): Section =
+    internal fun toSection(sectionDb: SectionDb, parentAccNo: String? = null): Section =
```

Comment:
I believe in the `SectionDb` and `ExtendedSection` parent section should be a reference to a section, not a plain string. AccNo is only a way for the user to indicate parent section but system internal should honor `parentId` .

Reply:
You're right about this. I removed this part since it's actually not necessary for this ticket. 

However, the `parentAccNo` is currently defined as a string in both `Section` and `ExtendedSeciton` models so, we may consider creating a separated chore task to refactor this.